python/nammu/controller/ConsoleController.py

```diff
@@ -27,7 +27,6 @@ class ConsoleController(object):
     def __init__(self, mainControler):
         # Load the config file
         self.config = mainControler.config
-        self.test = mainControler
```

Comment:
This is just removing an unused attribute of the class.

The change is logically unrelated to the rest of the pull request, but so small not to be really worth a pull request on its own